src/components/AChat/AChat.vue

```diff
@@ -42,31 +42,31 @@
   </div>
 </template>
```

Comment:
I'd suggest to update the class names in the template as well. See https://developers.adamant.im/template-classes.html